src/faq.md

```diff
@@ -79,6 +79,55 @@ outputs:
       outputEval: ${self[0].basename=inputs.otu_table_name; return self;}
 ```
 
+## Referencing a local script
+
+We are going to describe two methods of doing this:
+
+The first method involves [adding the folder containing your scripts to the `PATH` environment variable](https://stackoverflow.com/q/8779951/20123733). This allows you to run the shell script without using `sh` or `bash` commands
+
+
+Start with adding a _shebang_ at the top of your file:
+```
+#!/bin/bash
+```
+After that, make the script executable with command `chmod +x script.sh`
+
+Finally, modify your path to add the directory where your script is located (It is good practice to use `$HOME/bin` for storing your own scripts).
```

Comment:
s/path/PATH environment variable ?

Reply:
Could you please elaborate? I'm not sure of what exactly you mean here.